fpga_interchange/fasm_generators/xc7.py

```diff
@@ -452,11 +452,11 @@ def get_feature_prefix(site_thru_feature, wire):
                 regex="[LR]IOI_ILOGIC([01])_D",
                 features=["ZINV_D"],
                 callback=lambda m: "ILOGIC_Y{}".format(m.group(1))))
-        site_thru_features.append(ExtraFeatures(
-            regex="CLK_HROW_CK_MUX_OUT_([LR])([0-9]+)",
-            features=["IN_USE", "ZINV_CE"],
-            callback=lambda m: "BUFHCE.BUFHCE_X{}Y{}".format(0 if m.group(1) == "L" else 1, m.group(2))
-            ))
+        site_thru_features.append(
+            ExtraFeatures(regex="CLK_HROW_CK_MUX_OUT_([LR])([0-9]+)",
+                          features=["IN_USE", "ZINV_CE"],
+                          callback=lambda m: "BUFHCE.BUFHCE_X{}Y{}".format(
+                              0 if m.group(1) == "L" else 1, m.group(2))))
```

Comment:
This may be a leftover from the autoformatting error

Reply:
I've changed it to look inline with the rest of the file